docs/features/static_role_permissions.md

```diff
@@ -0,0 +1,130 @@
+# Static Role Permissions
+
+This library provides a highly simplified permission system
+based on a static (hardcoded) definition of roles and permissions.
+
+- This saves performance, because it does not require any database queries to check permissions.
+- This saves complexity, because the permissions are defined in a simple dictionary. No n:m relations, no changes at runtime.
+
+## Installation
+
+1. Add `StaticRolePermissionBackend` to your `AUTHENTICATION_BACKENDS` in the `settings.py`:
+
+```py
+AUTHENTICATION_BACKENDS = [
+    # ... other backends
+    # 'django.contrib.auth.backends.ModelBackend',  <-- replace the default ModelBackend
+    "ambient_toolbox.static_role_permissions.auth_backend.StaticRolePermissionBackend",
+]
+```
+
+2. Set `STATIC_ROLE_PERMISSIONS_PATH` as dotted path, pointing to a dictionary,
+   which defines your permissions per role.
+
+Example:
+
+`settings.py`
+```py
+STATIC_ROLE_PERMISSIONS_PATH = "my_project.conf.permisions.PERMISSIONS_DICT"
+```
+
+`my_project.conf.permisions.py`
+```py
+PERMISSIONS_DICT: dict[str, set[str]] = {
+    "role_1": {"permission_1", "permission_2", ...},
+    "role_2": {...},
+}
+```
+
+In fact, the roles and permissions can be any string, but it is recommended to follow the Django permission
```

Comment:
question: dont you check for permission existance? so you cant write everything here, right?

Reply:
If you disable the system check you can.

Reply:
👌